src/promptflow/promptflow/_sdk/_orm/experiment_node_run.py

```diff
@@ -0,0 +1,58 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from sqlalchemy import TEXT, Column
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.orm import declarative_base
+
+from promptflow._sdk._constants import EXP_NODE_RUN_TABLE_NAME
+from promptflow._sdk._errors import ExperimentNodeRunNotFoundError
+
+from .retry import sqlite_retry
+from .session import mgmt_db_session
+
+Base = declarative_base()
+
+
+class ExperimentNodeRun(Base):
+    __tablename__ = EXP_NODE_RUN_TABLE_NAME
+
+    snapshot_id = Column(TEXT, primary_key=True)
```

Comment:
why is this the primary key?  a snapshot can be used to run multiple runs?

Reply:
mark run_id as primary key